clamp.defaults.13.yaml

```diff
@@ -8,25 +8,31 @@ apache:
   options:
     servername: "{{$.address}}"
     listen: 80
-    pidfile: "{{$cwd}}/.clamp/tmp/httpd.pid"
-    lockfile: "{{$cwd}}/.clamp/tmp/accept.lock"
+    autoopen: false
+    errorlog: "{{$cwd}}/.clamp/logs/apache.error.log"
+    customlog: "{{$cwd}}/.clamp/logs/apache.access.log"
+    pidfile: "'{{$cwd}}/.clamp/tmp/httpd.pid'"
```

Comment:
There were single quotes in the original JSON file. I converted it to YAML and kept it.